src/components/Movies/MovieFilterBar.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC } from 'react'
+import classNames from "classnames"
+
+import styles from './MovieFilterBar.scss'
+
+const genres = ['All','Documentary','Comedy','Horror','Crime']
+
+const MovieFilterBar: FC = () => {
+  const activeGenre: string = 'All'
+  const activeSort: string = 'Release Date'
+
+  return (
+    <div className={styles['movieFilterBar']}>
+      <ul className={styles['movieFilterBar--genres']}>
+        {genres.map(genre => {
+          return (<li
+            className={classNames(
+              styles['movieFilterBar--genre'],
+              activeGenre === genre &&
```

Comment:
Although it's valid, why didn't you use an object? Like that: `classNames('foo', { bar: true });`